scripts/firezone-client.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Firezone Client
+
+[Service]
+Environment="RUST_LOG=firezone_linux_client=trace,wire=trace,connlib_client_shared=trace,firezone_tunnel=trace,connlib_shared=trace,warn" "FIREZONE_TOKEN=n.SFMyNTY.g2gDaANtAAAAJGM4OWJjYzhjLTkzOTItNGRhZS1hNDBkLTg4OGFlZjZkMjhlMG0AAAAkN2RhN2QxY2QtMTExYy00NGE3LWI1YWMtNDAyN2I5ZDIzMGU1bQAAACtBaUl5XzZwQmstV0xlUkFQenprQ0ZYTnFJWktXQnMyRGR3XzJ2Z0lRdkZnbgYAGUmu74wBYgABUYA.UN3vSLLcAMkHeEh5VHumPOutkuue8JA6wlxM9JxJEPE" "FIREZONE_API_URL=ws://api:8081" "FIREZONE_ID=D0455FDE-8F65-4960-A778-B934E4E85A5F"
+ExecStart=/home/user/projects/firezone/rust/target/debug/firezone-linux-client
```

Comment:
Is this actually the path that the binary will be in after you've moved it out of the container? Systemd seems to search the path (reference: https://unix.stackexchange.com/a/687292) so maybe we should use a relative path here or a well-known one such as `/usr/bin`?

Reply:
Good catch, that's a copy-paste from my dev system. In the CI runner it'll be somewhere else.